actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -334,12 +334,12 @@ def respond_to_missing?(*args)
             true
           end
 
-          def method_missing(called, *args, **options, &block)
+          def method_missing(called, *args, escape: true, **options, &block)
             name = called.to_s.dasherize
```

Comment:
```suggestion
            name = called.name.dasherize
```

Reply:
Looks like you got this in 946e46ebccaee862a3bed84d62424655723bc407